src/main/java/core/basesyntax/service/FileServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileServiceImpl implements FileService {
+    @Override
+    public List<String> read(String fileName) {
+        try {
+            return Files.readAllLines(Path.of(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file: " + fileName);
+        }
```

Comment:
Consider using try-with-resources for handling the file reading to ensure that all resources are closed properly.